python/lsst/ip/diffim/getTemplate.py

```diff
@@ -126,46 +126,51 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         butlerQC.put(outputs, outputRefs)
 
     def getOverlappingExposures(self, inputs):
-        """Return lists of coadds and their corresponding dataIds that overlap
-        the detector.
+        """Return a data structure containing the coadds that overlap the
+        specified bbox projected onto the sky, and a corresponding data
+        structure of their dataIds.
+        These are the appropriate inputs to this task's `run` method.
 
         The spatial index in the registry has generous padding and often
         supplies patches near, but not directly overlapping the detector.
-        Filters inputs so that we don't have to read in all input coadds.
+        This filters the inputs so that we don't have to read in all
+        possibly-matching coadds.
 
         Parameters
         ----------
         inputs : `dict` of task Inputs, containing:
             - coaddExposures : `list` \
                               [`lsst.daf.butler.DeferredDatasetHandle` of \
                                `lsst.afw.image.Exposure`]
-                Data references to exposures that might overlap the detector.
+                Data references to exposures that might overlap the desired
+                region.
             - bbox : `lsst.geom.Box2I`
-                Template Bounding box of the detector geometry onto which to
-                resample the coaddExposures.
+                Template bounding box of the pixel geometry onto which the
+                coaddExposures will be resampled.
             - skyMap : `lsst.skymap.SkyMap`
-                Input definition of geometry/bbox and projection/wcs for
-                template exposures.
+                Geometry of the tracts and patches the coadds are defined on.
             - wcs : `lsst.afw.geom.SkyWcs`
-                Template WCS onto which to resample the coaddExposures.
+                Template WCS onto which the coadds will be resampled, .
 
```

Comment:
an extra comma at the end of the -wcs explanation

Reply:
Also separate question: does this function only work on coadds, or is it possible this would be used for calexp files also (i.e. perhaps more generic reference to input images rather than specifically coadds would make sense?)

Reply:
Coadds are organized per tract/patch, other exposures are not. That tract/patch organization is a requirement here.